packages/mui-utils/src/isFocusVisible/isFocusVisible.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Returns a boolean indicating if the event's target has :focus-visible
+ */
+export default function isFocusVisible(event: React.FocusEvent): boolean {
```

Comment:
It feels a bit weird to call `isFocusVisible` on an event. A better API would be `hasFocusVisible(element: HTMLElement)`. You'll have to change call sites to `hasFocusVisible(event.target)`.

I realize it's meant to be temporary, but temporary code tends to live for years ;)

Reply:
Good call

Reply:
Refactored 👍🏼